core/helpers.cpp

```diff
@@ -497,8 +497,8 @@ bool SetRTPriorityRTKit(int prio)
         if(getrlimit(RLIMIT_RTTIME, &rlim) != 0)
             return errno;
 
-        TRACE("RTTime max: %llu (hard: %llu, soft: %llu)\n", umaxtime, ulonglong{rlim.rlim_max},
-            ulonglong{rlim.rlim_cur});
+        TRACE("RTTime max: %llu (hard: %llu, soft: %llu)\n", umaxtime, ulonglong{static_cast<ulonglong>(rlim.rlim_max)},
+            ulonglong{static_cast<ulonglong>(rlim.rlim_cur)});
```

Comment:
It's redundant to do both a plain `ulonglong{...}` cast and a `static_cast<ulonglong>(...)`. But what type is `rlim_t` on FreeBSD? An `unsigned long long` is the largest standard type, so it should be guaranteed to fit.

Reply:
it s signed on freebsd.